### PR TITLE
Feature/quarkus migration

### PR DESCRIPTION
**Pull request by fabricepipart**
This way we can discuss the migration and work together on it

### CODE REVIEW

**Comment by fabricepipart** [comment]
I noted the following TODOs:

- [ ] There is a warning regarding Jandex index and `org.jboss.logging.Logger` when running `mvn package -DskipTests`
- [ ] Unit Tests fail `mvn test`

Did you know about this?
https://t.co/qYuXt0k7FK

File: src/test/scripts/python/api/ping.py [hunk 1]
```diff
@@ -8,13 +8,13 @@
 
 
 # ----------------------------------------------------------------------
-def ping(dateTime, pilotBeaconId, power, chronoId):
+def ping(instant, pilotBeaconId, power, chronoId):
   "This adds a Ping"
-  data = '{"dateTime":"' + formatDatetime(dateTime) + '", "power":"' + str(power) + '"}'
+  data = '{"instant":"' + formatDatetime(instant) + '", "power":"' + str(power) + '"}'
```
Comment (fabricepipart) [inline src/test/scripts/python/api/ping.py]: You could have avoided a lot of pain by avoiding this name change!

File: src/test/java/org/trd/app/teknichrono/model/manage/TestLapTimeOrder.java [hunk 2]
```diff
@@ -292,13 +293,13 @@ public void fillGapsInfoWhenOrderedByDuration() {
       if (previousGap == -1) {
         Assert.assertEquals(0, l.getGapWithBest());
       }
-      if (l.getDuration() > 0) {
+      if (l.getDuration().compareTo(Duration.ZERO) > 0) {
         Assert.assertTrue(l.getGapWithBest() >= previousGap);
       } else {
         // No more gaps
-        previousGap = Long.MAX_VALUE;
+        previousGap = Long.MAX_VALUE; // FIXME this is a bug, always overridden
```
Comment (fabricepipart) [inline src/test/java/org/trd/app/teknichrono/model/manage/TestLapTimeOrder.java]: Can you explain the FIXME ?

File: src/test/scripts/python/api/ping.py (hunk 1 above)
Comment (loicrouchon) [reply src/test/scripts/python/api/ping.py]: Yes but instant is more meaningful in this context

File: src/test/java/org/trd/app/teknichrono/model/manage/TestLapTimeOrder.java (hunk 2 above)
Comment (loicrouchon) [reply src/test/java/org/trd/app/teknichrono/model/manage/TestLapTimeOrder.java]: write value was always overridden two lines later

**Comment by loicrouchon** [comment]
> I noted the following TODOs:
> 
>     * [ ]  There is a warning regarding Jandex index and `org.jboss.logging.Logger` when running `mvn package -DskipTests`
> 
>     * [ ]  Unit Tests fail `mvn test`
> 
> 
> Did you know about this?
> https://t.co/qYuXt0k7FK

I need to take a look at the Jandex index once it'll be working.

For the mvn tests, refactoring is on-going hence they are not yet fixed

**Pull request merged**